routes/messages-routes.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const process = require('process'); // process module which is used to get the current working directory of the node.js process
+const {v4 : uuidv4} = require('uuid')// import single method(v4) from uuid library and call it uuidv4.[ else uuid = require('uuid')  => uuid.v4()]
+let fs = require('fs');
+const router = express.Router();
+
+const currentMessagesJson = require("../messages.json");
+let currentMessagesTable = currentMessagesJson.messages;
+
+/////////////GET////////////////////////////
+
+router.get("/", (req, res) => { //http://localhost:8000/messages/
+  return res.status(200).json(currentMessagesJson);
+});
+
+router.get("/:uniq_id", (req, res) => { //http://localhost:8000/messages/88ab2e95-1955-482e-9107-bf2ae8825baf
+  const uniq_id = req.params.uniq_id;
+  return res.status(200).json({
+    message: currentMessagesTable.filter((message) => message.uniq_id === uniq_id)[0],
+  });
+});
+
+/////////////POST////////////////////////////
+
+router.post("/", (req, res) => { //http://localhost:8000/messages/
+  //const id = parseInt(currentMessagesTable[currentMessagesTable.length - 1].uniq_id) + 1; // and then id: String(id)
+  var newMessage = {text: req.body.text, uniq_id: uuidv4(), time: new Date().toLocaleString() ,user_name: req.body.user_name}; // Make new var tweet and 
+  currentMessagesTable.push(newMessage);
+
+  saveUpdatedMessagesToJsonFile(req, res);
+});
+
+/////////////PUT////////////////////////////
+
+router.put("/:uniq_id", (req, res) => { //http://localhost:8000/messages/88ab2e95-1955-482e-9107-bf2ae8825baf
+  const uniq_id = req.params.uniq_id;
+  const updatedMessageText = req.body.text;
+  const index = currentMessagesTable.map((message) => message.uniq_id).indexOf(uniq_id);
+  //console.log(currentMessagesTable.map((message) => message.uniq_id));
+  if(index !== -1) {
+    currentMessagesTable[index].text = updatedMessageText;
+    saveUpdatedMessagesToJsonFile(req, res);
+  }
+});
+
+/////////////DELETE////////////////////////////
+
+router.delete("/:uniq_id", (req, res) => { //http://localhost:8000/messages/88ab2e95-1955-482e-9107-bf2ae8825baf
+  const uniq_id = req.params.uniq_id;
+  const index = currentMessagesTable.map((message) => message.uniq_id).indexOf(uniq_id);
+  if(index !== -1) {
+    currentMessagesTable.splice(index, 1); // removes 1 element at the given index
+    saveUpdatedMessagesToJsonFile(req, res);
+  }
+});
+
+///////////////////////////////////////////////
+
+function saveUpdatedMessagesToJsonFile(req, res) {
```

Comment:
All the logic related to files etc should be in a different file. FileManager or something like this.
Also, you can create a kind of "abstract" file call it MessagesStore and it will wrap the file logic. Later on it will be changed to be mongo etc.